test/colony-network-mining.js

```diff
@@ -41,6 +41,10 @@ const contractLoader = new TruffleLoader({
 
 const useJsTree = true;
 
+const WAD = new BN(10).pow(new BN(18));
+const MIN_STAKE = WAD.muln(2000);
+const DEFAULT_STAKE = MIN_STAKE.muln(12);
```

Comment:
Can we increase this to say, 1000? Currently, tests that involve stakers that use `submitAndForwardTimeToDispute` advance 1800 seconds, then get all the clients to submit. With a stake of `MIN_STAKE.muln(12)`, there's a 1-in-4096 chance that any client passed in to that function will fail (because none of its entries are eligible yet after only 1800 seconds). We call that function 46 times in this file, so there's somewhere between a 1-in-45 to 1-in-90 (depending on how many clients are passed in to that function) that a test will randomly fail. By increasing this to `.muln(1000)`, which is effectively what it was before, those numbers are on the order of `10**300`, which I am much more comfortable with.

Reply:
100%

Reply:
And I still can't find the minimum stake hardcoded in `ReputationMiner.js`

Reply:
@area bump

Reply:
https://github.com/JoinColony/colonyNetwork/blob/84b8106ab8f124230e1c5fb6c758f683af946883/packages/reputation-miner/ReputationMiner.js#L535